### DISCUSSION TITLE
netclient for Android?

### DISCUSSION THREAD

**Discussion by danielkrajnik** [gravitl/netmaker#631] [General]
Would it be possible to create a netclient for Android? Was it ever considered? It's much better than ingress gateway, which creates another hub-and-spoke model. Is there any reason not to create one?

**Comment by afeiszli** [comment]
We have roadmapped adding a client app, but it is a ways off. We'll make a public roadmap soon which should include the intended timeframe for native phone apps.

**Reply by noseshimself** [reply]
Did anything change in regards to an Android client that is (ChromeOS!) available via the play store?

**Comment by mattkasun** [comment]
on the roadmap but not a high priority yet

**Comment by purepani** [comment]
Just wondering if there were any updates on the road map for this,

**Comment by mattkasun** [comment]
It is in active development, just need some polishing prior to formal release

**Reply by purepani** [reply]
Thanks, looking forward to it!

**Reply by archef2000** [reply]
Also, looking forward to it!

**Reply by noseshimself** [reply]
Please keep in mind reading up on ChromeOS requirements for Android applications to mark it correctly as VPN provider (and possibly a dozen other things -- I just noticed that applications working on most ChromeOS devices stopped working on the new "Chromebook Plus" subtype).

**Comment by Pheggas** [comment]
Hey. Is one released yet?

**Comment by theguy951357** [comment]
Our Remote Access Client is available for Android.

**Reply by Pheggas** [reply]
Ah, right! Thank you!
